crypto/ec/curve448/arch_64/arch_intrinsics.h

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2017-2018 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright 2016 Cryptography Research, Inc.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ *
+ * Originally written by Mike Hamburg
```

Comment:
It has been OpenSSL practice to list copyright holders and not individual contributors in the code, but in the commit. Does the project want to change that practice? It might open the floodgates.

Reply:
This is not the case. There are very many instances in the code where the original author is credited. Generally we only do this for the first version of a file and not for everyone that ever touches it.

Reply:
> This is not the case. There are very many instances in the code where the original author is credited. Generally we only do this for the first version of a file and not for everyone that ever touches it.

I looked via git grep for original and author.  I wouldn't call it many, and most of them are Mike in the 448 code.  Shrug.

Reply:
I grepped "written by" and got 175 hits (most of which were Andy). I stopped looking after that.

Reply:
ah, you win :)  thanks.